middle_end/simple_value_approx.ml

```diff
@@ -783,3 +783,55 @@ let float_array_as_constant (t:value_float_array) : float list option =
         | Value_extern _ | Value_boxed_int _ | Value_symbol _)
         -> None)
       contents (Some [])
+
+let potentially_taken_const_switch_branch t branch =
+  match t.descr with
+  | (Value_unresolved _
+    | Value_unknown _
+    | Value_extern _
+    | Value_symbol _) ->
+    (* In theory symbol cannot contain integers but this shouldn't
+       matter as this will always be an imported approximation *)
+    true
+
```

Comment:
It we are discussing style, I'd suggest to drop the custom `.ocp-indent\  in middle-end and adopt the global configuration.  I find the lack of extra indentation for pattern matching "actions" (w.r.t. the pattern) makes the code much harder to read.  I would also suggest to drop useless parentheses around or-patterns.